### ISSUE TITLE
Ability to get a list of available servers as a response from the mainserver.

### ISSUE THREAD

**Issue by Max98** [RigsOfRods/ror-server#4]
I've been looking at how to recode the multiplayer loby ingame, and the only little problem i have for now is that the list of the servers is just an html list which isn't going to be much of efficient for me. If i'm not mistaken, the WEBSERVER tick must be the main server where everything is registered? We should add a feature where the game would send a packet asking for all available servers, the map, and slots and of course, the IP of the target server.

**Comment by Hiradur** [comment]
Did you look into how RoRConfig does it?

**Comment by Max98** [comment]
RoRConfig just has a browser that connects to: http://api.rigsofrods.com/serverlist/ 
and renders it. Nothing interesting, Imo don't like it.

**Comment by Hiradur** [comment]
I'm not sure if I understand what you want. Only the master server has knowledge of all existing servers and we don't have control over it currently.

**Comment by Max98** [comment]
Oh so we don't have the master server's source code here? Well, that's not going to be easy.

**Comment by only-a-ptr** [comment]
@Max98 Situation has evolved. I got the old serverscripts, they were a mess, I began making new ones: https://github.com/only-a-ptr/multiplayer.rigsofrods.org. Since it has REST+JSON interface (as opposed to GET/plaintext), the server needs to be updated to work with it.

Currently it's a simple serverlist - just a database of server IP/port/status data. Should do the trick for now. For GET request, it outputs HTML for compatibility with RoRConfig. I'm planning to add JSON output for in-game serverlist (future)
